testing/databaseTest.py

```diff
@@ -415,7 +421,14 @@ def kinetics_checkAdjlistsNonidentical(self, database):
                         continue
 
                     nose.tools.assert_false(speciesList[i].molecule[0].isIsomorphic(speciesList[j].molecule[0], initialMap), "Species {0} and species {1} in {2} database were found to be identical.".format(speciesList[i].label,speciesList[j].label,database.label))
-
+    
+    def kinetics_checkLibraryRatesCanBeEvaluated(self, library):
+        """
+        This test ensures that every library reaction has evaluable kinetics
+        """
+        for entry in library.entries.values():
+            entry.data.getRateCoefficient(1000.0,1.0)
```

Comment:
Should we also add some additional checks here? For example, check that the rate is non-negative and within a reasonable range?